README.md

```diff
@@ -115,4 +115,13 @@ Please create a GitHub issue to report an issue or ask questions.
 ## Contributing
 
 Contributions to the OpenCL ICD Loader are welcomed and encouraged.
-You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request or other contribution to GitHub.
\ No newline at end of file
+You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request or other contribution to GitHub.
+
+## Table of Debug Environment Variables
+
+The following debug environment variables are available for use with the OpenCL ICD loader:
+
+| Environment Variable              | Behavior            |  Example Format      |
+|:---------------------------------:|---------------------|----------------------|
+| OCL_ICD_FILENAMES                 | Specifies a list of additional ICDs to load.  The ICDs will be enumerated first, before any ICDs discovered via default mechanisms. | `export OCL_ICD_FILENAMES=libVendorA.so:libVendorB.so`<br/><br/>`set OCL_ICD_FILENAMES=vendor_a.dll;vendor_b.dll` |
```

Comment:
In the ICD registry keys or vendor files we allow specifying paths. So I think we should support the same here and rename this variable to OCL_ICD_ENTRIES.

Reply:
This environment variable supports full paths to ICDs as well.  I can update the table to show this if it would be helpful; it only uses file names without full paths right now for brevity.  For example:

| ... | Example Format |
|--|--|
|... | `export OCL_ICD_FILENAMES=/path/to/libVendorA.so:libvendorB.so`

The name of the environment variable was purposefully chosen to be compatible with the Vulkan loader, so I'd prefer not to change it unless there is a very good reason to do so.

Reply:
I agree that it is good to be in sync with Vulkan loader.